.github/workflows/lint.yml

```diff
@@ -37,9 +37,6 @@ jobs:
       - name: Install dependencies
         run: |
           pip3 install demjson3
```

Comment:
Do we need this (and python itself) for any other reason? Or should I remove those?

Reply:
I think it supplies jsonlint, so I guess if you're killing things, be thorough.